tools/third_party.py

```diff
@@ -255,8 +255,7 @@ def download_clang_format():
 
 # Download clang by calling the clang update script.
 def download_clang():
-    run(['python',
-         tp('v8/tools/clang/scripts/update.py'), '--if-needed'],
+    run(['python', tp('v8/tools/clang/scripts/update.py')],
```

Comment:
No more  --if-needed?

Reply:
Dont think so / they seem to have removed it.